commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -65,11 +94,11 @@ trait UnicycleDrive extends Drive {
   import UnicycleControllers._
 
   object unicycleTasks {
-    class ContinuousDrive(forward: Signal[Dimensionless], turn: Signal[Dimensionless])
-                         (implicit drive: Drivetrain) extends ContinuousTask {
+    class ContinuousClosedDrive(forward: Signal[Dimensionless], turn: Signal[Dimensionless])
+                               (implicit drive: Drivetrain) extends ContinuousTask {
       override def onStart(): Unit = {
         val combined = forward.zip(turn).map(t => UnicycleSignal(t._1, t._2))
-        drive.setController(toClosedDrive(combined))
+        drive.setController(parentClosedLoop(combined))
```

Comment:
I don't understand the whole purpose of this parentClosedLoop thing. What is the point, and what does it do?